src/utils/identity/ledger/simulated.ts

```diff
@@ -0,0 +1,52 @@
+import { PublicKey } from '@dfinity/agent';
+import { blobFromBuffer } from '@dfinity/candid';
+import { Principal } from '@dfinity/principal';
+import DfinityApp from '@zondax/ledger-icp';
+
+import Secp256k1PublicKey from '../secpk256k1/publicKey';
+import { Buffer as BlobBuffer } from '../../../../node_modules/buffer';
+import LedgerIdentity from '.';
+
+class SimulatedLedgerIdentity extends LedgerIdentity {
```

Comment:
what is this for???

Reply:
For testing purpose, to be able to use Simulates Ledger